Core/Model/AtributoValor.php

```diff
@@ -71,30 +71,90 @@ public function primaryColumn()
         return 'id';
     }
 
-    public function install()
+    /**
+     * Returns the name of an attribute
+     *
+     * @return string
+     */
+    public function getNombre()
     {
-        new Atributo();
+        $nombre = '';
 
-        return '';
+        $sql = 'SELECT * FROM atributos WHERE codatributo = ' . $this->dataBase->var2str($this->codatributo) . ';';
+        $data = $this->dataBase->select($sql);
+        if (!empty($data)) {
+            $nombre = $data[0]['nombre'];
+        }
+
+        return $nombre;
     }
 
-    public function test()
+    /**
+     * Store the model data in the database.
+     *
+     * @return bool
+     */
+    public function save()
     {
         $this->valor = self::noHtml($this->valor);
-        return true;
+
+        return $this->saveTrait();
     }
 
     /**
-     * Selecciona todos los atributos de un código de atributo
+     * Select all attributes of an attribute code
      *
      * @param string $cod
      *
      * @return self[]
      */
     public function allFromAtributo($cod)
     {
-        $where = [new DataBaseWhere('codatributo', $cod)];
```

Comment:
Esta función debe ser:
$where = [new DataBaseWhere('codatributo', $cod)];
        $order = ['valor' => 'ASC'];
        return $this->all($where, $order);